src/organizations/components/OrgProfileDeletePanel.tsx

```diff
@@ -65,8 +65,8 @@ const OrgProfileTab: FC = () => {
         {CLOUD && quartzMe?.accountType === 'free' && (
           <>
             <FlexBox.Child>
-              <h4 style={{marginBottom: '4px'}}>Delete Organization</h4>
-              <p style={{margin: '0px'}}>
+              <h4>Delete Organization</h4>
+              <p style={{marginTop: '16px', marginBottom: '8px'}}>
```

Comment:
preference for classes using clockface variables for consistency

Reply:
done